crates/sncast/src/lib.rs

```diff
@@ -48,6 +48,7 @@ struct Account {
     salt: Option<String>,
     deployed: Option<bool>,
     class_hash: Option<String>,
+    legacy: Option<bool>,
```

Comment:
I'm afraid that the definition of legacy may be too fluid for the future. Maybe we should just hold cairo version in here?

Reply:
This way, it aligns with the keystore account files, making it easier to use the same `Account` struct. Moreover, differentiating between Cairo 1 and 2 accounts seems non-trivial (please let me know if there a simple to way to achieve this), and it's not relevant in terms of encoding.

Reply:
ok, makes sense. guess we will have to wait and see how thing will look like with newer cairo versions then.